linera-service/src/database_tool.rs

```diff
@@ -165,9 +155,9 @@ async fn main() -> Result<(), anyhow::Error> {
                 .await
                 .expect("successful delete_all operation");
```

Comment:
Can we use `?` consistently? The panic will return error code 101 by the way.

Reply:
Yes, you are right.